extra/import_data.py

```diff
@@ -1,197 +1,158 @@
 #!/usr/bin/env python3
 
-"""
-Usage: Import Alissa export in the form of .xsls into a MongoBD database.
-       Make sure the data is in a sub folder called 'input'.
-Python version: 3.7.1
-
-Input .xlsx format
-[Variant List] - spans 1A - 1E
-[General Metadata] -
-    - header spans 2A - 8B
-    - value spans 2C - 8E
-[Annotation Sources]
-    - header spans 9A - 9B
-    - subheader spans 9C - 22C
-    - value spans 9D - 22H
-[Empty line]
-[Data Table of unknown length with first row as header]
-"""
-
 import os
-import io
 import time
-import pymongo
-import warnings
-import progressbar
-import argparse
 import sys
+import pymongo
 import logging
 import logging.config
 import yaml
-from openpyxl import load_workbook
+import config
+import dateutil.parser
+from requests.exceptions import HTTPError
+from alissa_interpret_client.alissa_interpret import AlissaInterpret
+from datetime import datetime, timezone
+
+# Create Alissa connection based on config.py (fill details there)
+client = AlissaInterpret(
```

Comment:
maybe rename client to `alissa_client`, like `mongo_client`?

Reply:
can be done if that makes it more clear.